src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -44,6 +44,34 @@ class databaseInfo extends resourceInfo {
         this.documentsCount(databaseInfo.DocumentsCount);
         this.indexesCount(databaseInfo.IndexesCount);
     }
+
+    static empty(resourceName: string): databaseInfo {
```

Comment:
Can we avoid having this method? Instead when we get info that new resource was created, then fetch information only for this resource and then add to resources list? I think @Danielle9897 so implementing something similar.